content/blog/2024/04/camel-k-2-3/index.md

```diff
@@ -0,0 +1,66 @@
+
+First of all, we have changed the default runtime to Camel K Runtime version 3.8.1 which is based on same Camel Quarkus version (and Camel 4.4.1 version). It follows a list of the most important features you'll be able to use from now on.
+
+## Multi architecture manifest (ARM64)
+The possibility to run an ARM64 based platform was already available since version 2.0. However, the procedure to run the new architecture platform was a bit clumsy as you had to declare most of the things manually. In this version we have published a manifest that will contain the various architecture we support (AMD64 and ARM64). Your cloud provider will be able to pick the right platform from the manifest, freeing you up from specifying the platform.
+
+In order to maintain backward compatibility we still had to let the user declare which is the platform to use introducing the `builder.platform` trait configuration. Configure this parameter either in each Integration or in general into the IntegrationPlatform in order to specify which platform you want to run for all your Integrations.
+```
+kamel run test.yaml -t builder.platforms=linux/arm64 -t builder.platforms=linux/amd64
+
+```
+
+## Add support for Strimzi Kafka
+In this release we've dedicated a bit of time to enhance the user experience when it comes to use Kafka into your Integrations. From now on you'll be able to use Strimzi Kafka Custom Resource out of the box in a Pipe declaration. Here a brief example:
+```
+apiVersion: camel.apache.org/v1
+kind: Pipe
+...
+spec:
+  source:
+...
+  sink:
+    ref:
+      apiVersion: kafka.strimzi.io/v1beta2
+      kind: Kafka
+      name: my-cluster
+    properties:
+      topic: "my-topic"
+```
+
+## Polish Integration conditions
+Running a software in Kubernetes always come with the requirement to be able to observe and understand what's going on. We've worked on a series of enhancement to include more custom resources conditions and let you be able to inspect what's going on on an Integration or the Build which have started. Just to name a few of them we've included the list of Integration and IntegrationKit traits execution, those warning or deprecation notices that a trait used to log on operator log only and a Build waiting condition (to let you know that the Build is queued for a given reason).
+
+Hopefully these information are helping your SRE duties.
+
+## Publishing strategy
+In the previous Camel K versions we had deprecated certain supported publishing strategy. We have decided to remove them in this version in order to have an easier maintenance. We have also decided to default Jib as publishing strategy for plain Kubernetes providers (S2I is still the default of Openshift). As we know this may potentially affect some user out there, we've provided a guideline to [adopt your own publishing strategy](/camel-k/next/pipeline/pipeline.html#build-pipeline-examples-buildah).
```

Comment:
```suggestion
In the previous Camel K versions we had deprecated certain supported publishing strategy. We have decided to remove them in this version in order to have an easier maintenance. We have also decided to default [Jib](https://github.com/GoogleContainerTools/jib) as publishing strategy for plain Kubernetes providers (S2I is still the default of OpenShift). As we know this may potentially affect some user out there, we've provided a guideline to [adopt your own publishing strategy](/camel-k/next/pipeline/pipeline.html#build-pipeline-examples-buildah).
```